client/dataEntry.js

```diff
@@ -31,20 +31,12 @@ Template.dataEntry.helpers({
   },
 
   isSelectedRoundClosed: function() {
-    if(!this.roundId) {
-      // If there's no round selected, then the selected round is definitely
-      // *not* closed =)
-      return false;
-    }
-    return Rounds.findOne(this.roundId).isClosed();
+    round = Rounds.findOne(this.roundId);
```

Comment:
Hmm... my paranoia here comes from a few places:

1. Direct database modifications. This is not something I think we should actually worry about, although there is at least one place in our codebase where we mutate the database with validations turned off.
2. SimpleSchema/whatever doesn't handle validations correctly when you only update a single field (as you mentioned in a comment somewhere in this diff). I think this is a failing of SimpleSchema, and shouldn't rule how we design our system.
3. Bugs/database migrations that introduce/remove constraints on our database. This is my biggest source of paranoia, and I believe is pretty valid. At some point, there's going to be a bug that allows data inside that we don't actually trust. (Just to be clear, I don't mean this as a security concern, just as a correctness thing).

That all said, at times I have found myself about to write code like the following:

```
a = 1
a++
assert a == 2
```

These occassions have always been sources of great WTF-am-I-doing.

In this particular case, even though we may trust `this.roundId` (I'm not actually sure where it came from, but it's quite believable that it came from the database), IMO it doesn't make the code any dirtier to write it in a way that is a little more strict about it. In fact, the ternary I suggested is 1 line to the current 2 lines.